docs/guides/platform/Install_MERLIN_In_AirGap.md

```diff
@@ -164,131 +175,163 @@ docker run --name mirror-registry -p <the_registry_host_port>:5000 \
-```
-firewall-cmd --add-port=<the_registry_host_port>/tcp --zone=internal --permanent
 
+```bash
+firewall-cmd --add-port=<the_registry_host_port>/tcp --zone=internal --permanent
 firewall-cmd --add-port=<the_registry_host_port>/tcp --zone=public --permanent
-
 firewall-cmd --reload
 ```
+
 If a self-signed certificate is being use, add it to the list of trusted certificates.
-```
-cp /opt/registry/certs/domain.crt /etc/pki/ca-trust/source/anchors/
 
+```bash
+cp /opt/registry/certs/domain.crt /etc/pki/ca-trust/source/anchors/
 update-ca-trust
 ```
+
 Verify that the registry is available.
-```
+
+```bash
 curl -u <registry_user_name>:<registry_password> -k https://<the_registry_host_name>:<the_registry_host_port>/v2/_catalog
 ```
+
 See these parameter descriptions:
-```
+
+```text
 registry_user_name is the username to access the registry.
 registry_password is the password of the registry user.
 the_registry_host_nameis the registry domain name that was specified in the certificate. For example, registry.example.com.
 the_registry_host_port is the port that the Docker registry uses to serve content.
 ```
+
 Following is a sample response:
-```
+
+```text
     {"repositories":[]}
 ```
+
 Create a Docker certs folder and place the certificate in the folder.
-```
+
+```bash
 mkdir -p /etc/docker/certs.d/<the_registry_host_name>:<the_registry_host_port>
 cp /opt/registry/certs/domain.crt /etc/docker/certs.d/<the_registry_host_name>:<the_registry_host_port>/ca.crt
 ```
+
 Log in the Docker registry.
-```
+
+```bash
 docker login <the_registry_host_name>:<the_registry_host_port> -u <registry_user_name> -p <registry_password>
 ```
 
 #### Configure the registry
 
 After creates the registry, configure the Docker registry:
 
-- Create registry namespaces.
-- Create a separate registry namespace for each public registry source.
-* cpopen - Namespace to store all Operator images from the icr.io/cpopen namespace.
-* cp/ibmi-merlin - Namespace to store the IBM images from the cp.icr.io/cp/ibmi-merlin repository. The cp/ibmi-merlin namespace is for the images in the IBM Entitled Registry that require a product entitlement key and credentials to pully.
-
-- Verify that each namespace meets the following requirements:
-* Supports auto-repository creation.
-* Has credentials of a user who can write and create repositories. The bastion host uses these credentials.
-* Has credentials of a user who can read all repositories. The OpenShift Container Platform cluster uses these credentials.
+* Create registry namespaces.
+* Create a separate registry namespace for each public registry source.
+  * cpopen - Namespace to store all Operator images from the icr.io/cpopen namespace.
+  * cp/ibmi-merlin - Namespace to store the IBM images from the cp.icr.io/cp/ibmi-merlin repository. The cp/ibmi-merlin namespace is for the images in the IBM Entitled Registry that require a product entitlement key and credentials to pull.
```

Comment:
For readability, might be good to wrap these names in back ticks like `cpopen`, `icr.io/cpopen`, `cp.icr.io/cp/ibmi-merlin`, `cp/ibmi-merlin`, etc.

May need to check for similar names on this page or other pages.

Reply:
@SanjulaGanepola Changes made in 6d0afe6